### PR TITLE
Added implied conversion rate to Venmo and Revolut deposit creation t…

### PR DESCRIPTION
**Pull request by bweick**
…able.

image

Screen Shot 2024-05-06 at 10 34 57 AM

### CODE REVIEW

**Review by asoong — APPROVED** [review]
LGTM. Any reason not to add to UPI / Garanti?

**Comment by bweick** [comment]
> LGTM. Any reason not to add to UPI / Garanti?

can def add it

File: client/src/components/Swap/index.tsx [hunk 1]
```diff
@@ -474,6 +480,9 @@ const SwapForm: React.FC<SwapFormProps> = ({
       case QuoteState.INSUFFICIENT_LIQUIDITY:
         return 'Insufficient liquidity';
 
+      case QuoteState.AMOUNT_BELOW_TRANSFER_MIN:
+        return 'Send Amount Less Than Minimum';
```
Comment (asoong) [inline client/src/components/Swap/index.tsx:484]: Lower case the error string except the first word

File: client/src/helpers/strings/revolut.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ const strings: PlatformStrings = {
 
   // Notarizations Instructions
   NO_NOTARIZATIONS_ERROR: `
-    No Revoluttag details found.
+    No Revolut Tag details found.
```
Comment (asoong) [inline client/src/helpers/strings/revolut.ts:25]: Whoops meant to put `Revtag` in there, we do that everywhere for consistency

File: client/src/components/Deposit/revolut/NewPosition.tsx [hunk 3]
```diff
@@ -402,6 +402,15 @@ export const NewPosition: React.FC<NewPositionProps> = ({
     }
   }, [usdcBalance, isLoggedIn]);
 
+  const conversionRateLabel =  useMemo(() => {
+    if (isLoggedIn && depositAmountInput && receiveAmountInput) {
+      // Flipping conversion rate (i.e. relative to Venmo) b/c EUR/USD is quoted with USD as the base currency
+      return `Rate: ${calculateConversionRate(receiveAmountInput, depositAmountInput)} EUR/USDC`;
```
Comment (asoong) [inline client/src/components/Deposit/revolut/NewPosition.tsx:408]: @richardliang heads up we'll have to address this in multi currency

**Review by asoong — CHANGES_REQUESTED** [review]
Looking mostly good, minor nits

**Review by asoong — APPROVED** [review]
LGTM

**Pull request merged**